NomadAudio/test/SampleRateConverterTest.cpp

```
@@ -0,0 +1,421 @@
+
+void testQualityLevels() {
+    std::cout << "\n=== Test: All Quality Levels ===\n";
+    
+    const SRCQuality levels[] = {
+        SRCQuality::Linear,
+        SRCQuality::Cubic,
+        SRCQuality::Sinc8,
+        SRCQuality::Sinc16,
+        SRCQuality::Sinc64
+    };
+    const char* names[] = {"Linear", "Cubic", "Sinc8", "Sinc16", "Sinc64"};
```

Clang (14.0.6)

[warning] do not declare C-style arrays, use std::array<> instead

(cppcoreguidelines-avoid-c-arrays,modernize-avoid-c-arrays)